src/rs/scramble/scramble_search.rs

```diff
@@ -56,17 +56,20 @@ pub(crate) fn basic_idfs(
 pub(crate) fn filtered_search(
     scramble_pattern: &KPattern,
     generators: Generators,
-    min_optimal_moves: Option<usize>,
-    min_scramble_moves: Option<usize>,
+    min_optimal_moves: usize,
+    min_scramble_moves: Option<usize>, // inclusive
+    max_scramble_moves: Option<usize>, // exclusive
```

Comment:
I'd like to rename these at some point to be more clear, but I added comments for now.

Reply:
Actually, it might be less confusing to just add 1 when passing to the second IDFS, I'll do that.